src/core/initialization.jl

```diff
@@ -0,0 +1,109 @@
+const RELAXED_FORMULATION_MAPPING = Dict(
+    :ThermalStandard => DeviceModel(PSY.ThermalStandard, ThermalBasicUnitCommitment),
+    :ThermalMultiStart =>
+        DeviceModel(PSY.ThermalMultiStart, ThermalBasicUnitCommitment), # Compact vs Standard representation
+    :HydroDispatch => DeviceModel(PSY.HydroDispatch, HydroDispatchRunOfRiver),
+    :HydroEnergyReservoir =>
+        DeviceModel(PSY.HydroEnergyReservoir, HydroDispatchRunOfRiver),
+    :HydroPumpedStorage =>
+        DeviceModel(PSY.HydroPumpedStorage, HydroDispatchPumpedStorage),
+    :RenewableFix => DeviceModel(PSY.RenewableFix, FixedOutput),
+    :RenewableDispatch => DeviceModel(PSY.RenewableDispatch, RenewableFullDispatch),
+    :GenericBattery => DeviceModel(PSY.GenericBattery, BookKeeping),
+    :BatteryEMS => DeviceModel(PSY.BatteryEMS, BookKeeping),
+    :TapTransformer => DeviceModel(PSY.TapTransformer, StaticBranch),
+    :Transformer2W => DeviceModel(PSY.Transformer2W, StaticBranch),
+    :MonitoredLine => DeviceModel(PSY.MonitoredLine, StaticBranchUnbounded),
+    :Line => DeviceModel(PSY.Line, StaticBranch),
+    :HVDCLine => DeviceModel(PSY.HVDCLine, HVDCDispatch),
+    :PowerLoad => DeviceModel(PSY.PowerLoad, StaticPowerLoad),
+    :InterruptibleLoad => DeviceModel(PSY.InterruptibleLoad, InterruptiblePowerLoad),
+    :VariableReserve => ServiceModel(PSY.VariableReserve{PSY.ReserveUp}, RangeReserve),
+    :ReserveDemandCurve =>
+        ServiceModel(PSY.ReserveDemandCurve{PSY.ReserveUp}, StepwiseCostReserve),
+)
+
+function _build_initialization_template(problem::OperationsProblem)
+    ic_template = OperationsProblemTemplate(problem.template.transmission)
+    for (device, model) in problem.template.devices
+        model = RELAXED_FORMULATION_MAPPING[device]
+        set_device_model!(ic_template, model)
+    end
+    for (device, model) in problem.template.branches
+        model = RELAXED_FORMULATION_MAPPING[device]
+        set_device_model!(ic_template, model)
+    end
+    for (device, model) in problem.template.services
+        model = RELAXED_FORMULATION_MAPPING[device]
+        set_service_model!(ic_template, model)
+    end
+    return ic_template
+end
+
+function _build_initialization_problem(
+    problem::OperationsProblem{M},
+    sim::Simulation,
+) where {M <: AbstractOperationsProblem}
+    settings = deepcopy(get_settings(problem))
+    set_horizon!(settings, 1)
+    template = _build_initialization_template(problem)
+    ic_op_problem = OperationsProblem{M}(template, problem.sys, settings)
+    build!(ic_op_problem, output_dir = get_internal(problem).output_dir)
```

Comment:
we should pass the flags needed to avoid serialization of the system and the problem.